README.md

```diff
@@ -73,13 +73,13 @@ replacing `docker run swarm` with just `swarm`.
 
 ## Participating
 
-You can contribute to Docker Swarm in several different ways:
+You can [contribute](https://github.com/docker/swarm/blob/master/CONTRIBUTING.md) to Docker Swarm in several different ways:
```

Comment:
Just a nit; I think this can be replaced by simply `CONTRIBUTING.md` (it's in the same repository)